pagopa/io-app
# fix: [IOBP-710] First payment onboarding

## Short description
This PR fixes an issue related to onboarding: If you were onboarding a method for the first time and no past transactions were present either, it would not properly refresh onboarded methods when redirected to the homepage

## List of changes proposed in this pull request
- Added a dispatch action to refresh the onboarded method after an onboarding with success

## How to test
Try to delete all your onboarded payment methods and all the transactions, if you start a payment method onboarding you should be able to see the onboarded method when redirected to the payments homepage
